packages/demos/src/node-integrator.ts

```diff
@@ -1,6 +1,6 @@
 import {Css} from '@feature-hub/react';
 import express from 'express';
-import webpack from 'webpack';
+// import webpack from 'webpack';
```

Comment:
Typings not working with webpack 5 any more

Reply:
we need https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50802 so that Webpack 5 types are compatible with the webpack-dev-middleware types